rollup/core/objects/RollupState__c/fields/RelatedRecordKeys0__c.field-meta.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>RelatedRecordKeys0__c</fullName>
+    <description>Comma separated list of parent records with any rollup state associated with them</description>
+    <label>Related Record Keys</label>
```

Comment:
An incredibly small thing: should the label include `0` to be consistent with the other field labels? Or do you prefer not having the number on this one?

```suggestion
    <label>Related Record Keys 0</label>
```

Reply:
Oh definitely! I think this is a holdover from when the first field didn't have a number, and I missed updating it when I changed it to 0